pxr/imaging/plugin/hdRpr/CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ pxr_plugin(hdRpr
 		
 		${OptIncludeDir}
 				
-    PUBLIC_CLASSES
+    PRIVATE_CLASSES
 		rendererPlugin
         renderDelegate

```

Comment:
is the indentation weird here?

Reply:
I try to not fix code style issues in such commits. Otherwise, it would be much bigger commits

Reply:
I will create one big PR with code style fixes when we are done with all current PRs